### PR TITLE
Document changes about `/packit rebuild-failed` and `retest-failed`

### PR DESCRIPTION
**Pull request by nikromen**
Merge after [packit/packit-service#1303](https://github.com/packit/packit-service/pull/1303)

### CODE REVIEW

File: content/docs/packit-service.md [hunk 1]
```diff
@@ -90,6 +90,14 @@ for the particular check:
 
 ![Re-run Github check](/github-check-rerun.png)
 
+Or it is possible to re-trigger every failed task using pull request command
```
Comment (jpopelka) [inline content/docs/packit-service.md:93]:
```suggestion
Or it is possible to re-trigger every failed task using a pull request comment
```

File: content/docs/packit-service.md [hunk 2]
```diff
@@ -90,6 +90,14 @@ for the particular check:
...
+
+    /packit rebuild-failed
+
+which builds only against failed builds and similar for testing farm
+
+    /packit retest-failed
+
```
Comment (jpopelka) [inline content/docs/packit-service.md:100]:
```suggestion
which builds only failed builds and similar for testing farm

    /packit retest-failed

to <explicitly say what it does>
```

**Review by TomasTomecek — APPROVED** [review]
LGTM after incorporating @jpopelka's suggestions :)

File: content/docs/packit-service.md (hunk 2 above)
Comment (jpopelka) [reply content/docs/packit-service.md:100]: you left the `against`, which I'd suggested to remove. Did you overlook it or am I missing something?

Comment (nikromen) [reply content/docs/packit-service.md:100]: True. Thanks, Jirko!

**Review by jpopelka — APPROVED** [review]
:+1:

**Pull request merged**